agent/README.md

```diff
@@ -1,38 +1,3 @@
-| DEFAULT_REQUESTS_MEMORY | *self explanatory*                                                                                        | 64Mi                                   |
-| DEFAULT_VOLUME_SIZE     | Default volume size to use when not defined                                                               | 1G                                     |
-| DEFAULT_TIMEOUT         | timeout with GRPC connections, format is time.Duration                                                    | 5s                                     |
-| GIN_MODE                | Using release is fine, debug is for development                                                           | release                                |
-| GRPC_ADDRESS            | Backend GRPC address                                                                                      | localhost:5000                         |
-| GRPC_INSECURE           | Use GRPRC without TLS (for development)                                                                   | true                                   |
-| GRPC_KEEPALIVE          | GRPC keepalive interval, format is time.Duration                                                          | 60s                                    |
-| PORT                    | HTTP port to use                                                                                          | 8080                                   |
-| IMPORT_CONTAINER_IMAGE  | Import container name                                                                                     | rclone/rclone:1.57.0                   |
-| INGRESS_ROOT_DOMAIN     | Domain name for Ingress                                                                                   | *none*                                 |
-| NODE_ID                 | ID of the node                                                                                            | cb7e9573-9a43-4d5b-8005-eb8bb7a423c4"` |
-| READ_HEADER_TIMEOUT     | Gin read header timeout, configured to protect against Slowloris type attacks (form of denial of service) | 15s                                    |
-| REGISTRY                | Default registry address                                                                                  | index.docker.io                        |
-| REGISTRY_PASSWORD       | Password of the remote registry where DAgent is located                                                   | *none*                                 |
-| REGISTRY_USERNAME       | User of the remote registry where DAgent is located                                                       | *none*                                 |
+# dyrector.io platform: agent
 
+We are working on the source code documentation until, please use the root [README.md](../../README.md) for further information or check our official [documentation](https://docs.dyrector.io/) site.
```

Comment:
`We are working on the source code documentation, until then please use the root ...` sounds a bit better in my opinion 😇

Reply:
Fixed.